client/src/components/Home/Home.js

```diff
@@ -35,24 +47,33 @@ export const Home = () => {
     setIsLoading(false);
   };
 
-  const handleFilterCategory = (e) => {
+  
+     const nextPage = () => {
+      setSkip(skip + limit)
+    }
+
+    const previousPage = () => {
+      setSkip(skip - limit)
+    }
+
+  const handleFilterCategory = async (e) => {
     let value = e.target.value;
-    let result = [];
-    result = classes.filter((data) => {
-      return data.category.search(value) != -1;
-    });
-    setFilteredClasses(result);
+    setQueryString(queryString.concat(`&category=${value}`));
+    setCat(value);
   };
 
   const handleFilterCity = (e) => {
     let value = e.target.value;
-    let result = [];
-    result = classes.filter((data) => {
-      return data.meetingAddress.city.search(value) != -1;
-    });
-    setFilteredClasses(result);
+    setQueryString(queryString.concat(`&city=${value}`));
+    setCity(value);
   };
 
+    const handleFilterDate = (e) => {
+    setQueryString(queryString.concat(`&date=${e}`));
+    localStorage.setItem("date", e.target.value)
```

Comment:
Why are you using the local storage here? What is the useCase?

Reply:
Removed, was left from first implementation version